blerry/blerry.be

```diff
@@ -360,6 +361,7 @@ class Blerry_Device
   var next_forced_publish
 
   def init(mac, config, blerry_inst)
+    self.name = config.find('name', config['alias'])
```

Comment:
This line means "if name is not in the config, use the alias as the name", yeah? Can you adjust the comment you added to the readme to clarify?

Reply:
correct. Yea I can adjust the comment

Reply:
This should not change behavior for any existing users